db/migrate/20151021194020_add_time_of_day_to_scheduled_messages.rb

```diff
@@ -0,0 +1,5 @@
+class AddTimeOfDayToScheduledMessages < ActiveRecord::Migration
+  def change
+    add_column :scheduled_messages, :time_of_day, :time, null: false, default: "12:00:00"
```

Comment:
@jessieay:  Despite using both `default` (which should be the correct one to use) and `selected` as options for the form field in the employee form, I cannot get it to recognize using the `Eastern Time (US & Canada)` option as the default.  Do you have any insight into why this might be?